german/words/bei.md

```diff
@@ -0,0 +1,26 @@
+# bei 
+## Type 
+- _**Präposition**_ 
+## Pronunciation 
+- _**baɪ̯**_ 
+## Meaning 
+- **:''mit Dativ''** 
```

Comment:
this should be omitted, see https://de.wiktionary.org/wiki/bei . Blocking for now.